### DISCUSSION TITLE
Prevent export LDFLAGS and CPPFLAGS

### DISCUSSION THREAD

**Discussion by vinkovsky** [Homebrew/discussions#1458] [Everyday usage]
I was trying to use homebrew on my Mac yesterday. After installation. several packages i saw a similar message in the terminal
```
If you need to have sqlite first in your PATH, run:
  echo 'export PATH="/opt/homebrew/opt/sqlite/bin:$PATH"' >> ~/.zshrc

For compilers to find sqlite you may need to set:
  export LDFLAGS="-L/opt/homebrew/opt/sqlite/lib"
  export CPPFLAGS="-I/opt/homebrew/opt/sqlite/include"
```

So I did not hesitate to follow all these recommendations for each package, naturally I didn’t have an idea of what it was and why. After I have done a lot of things, please explain to me what this procedure is for and how obligatory it is. If it is not necessary or, on the contrary, harmful, how can I roll back these changes?

**Comment by carlocab** [comment]
If you don't know whether you need it or not, then you probably don't need it. You can just remove the relevant lines from your `~/.zshrc`.

**Reply by vinkovsky** [reply]
Thanks for the answer @carlocab ! Where are locate this strings? 

```
export LDFLAGS="-L/opt/homebrew/opt/sqlite/lib"
  export CPPFLAGS="-I/opt/homebrew/opt/sqlite/include"
```

**Reply by carlocab** [reply]
Does `echo $LDFLAGS $CPPFLAGS` return anything? If not, then you don't need to locate them, as they were never set to persist across terminal sessions. (If you just copied and pasted the suggested command from `brew` then they will not be persistent.)

**Reply by vinkovsky** [reply]
Thanks @carlocab! These variables do not return anything. I think the question can be closed
